doc/architecture/backend.md

```diff
@@ -0,0 +1,68 @@
+# Backend Architecture
+
+## Code documentation
+
+The code documentation of the backend can be built using `cargo doc --open`.
+
+You can find a more detailed explanation of which modules do what there.
+
+## API documentation
+
+The API documentation can be viewed by running the backend with `cargo run` and then navigating to <http://localhost:8080/doc/api/swagger/ui/>.  
+It is automatically built using [utoipa](https://github.com/juhaku/utoipa).
```

Comment:
This should be rather at the end. Start with structure to give an overview what it is about.